### PR TITLE
Initial commit of tools directory

### PR DESCRIPTION
**Pull request by wdhawkins**
Adding `tools` directory.  This initial commit includes:

`developer/lsan.supp`: lsan suppressions file
`xs/OpenMC_Create_MGXS_Library.ipynb`: Jupyter notebook for creating OpenMC cross-section libraries

### CODE REVIEW

**Comment by ragusa** [comment]
should we have a readme in the tools directory explaining what is found in its subdirectory? or a readme pointing to readme's in each subdirectory and those readme should each have one bullet describing their use/purpose?

File: tools/xs/OpenMC_Create_MGXS_Library.ipynb
Comment (zhardy-lanl) [inline tools/xs/OpenMC_Create_MGXS_Library.ipynb:1]: My only comment here would be to ask whether we want to stick to file naming conventions here or not. I would suggest modifying the name to `create_openmc_xslib.ipynb` (even if sticking to caps).

**Review by zhardy-lanl — APPROVED** [review]

File: tools/xs/OpenMC_Create_MGXS_Library.ipynb
Comment (andrsd) [reply tools/xs/OpenMC_Create_MGXS_Library.ipynb:1]: I think we should stick to the naming conventions, i.e. `create_openmc_xslib.ipynb`

Comment (wdhawkins) [reply tools/xs/OpenMC_Create_MGXS_Library.ipynb:1]: Fixed.

**Review by andrsd — APPROVED** [review]

File: tools/README.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This directory contains various tools for use by both devleopers and users.
```
Comment (ragusa) [inline tools/README.txt:1]: typo

**Review by ragusa — APPROVED** [review]
can you change the readme.txt to a markdown readme? if not, i'll do it soon

**Review by ragusa — CHANGES_REQUESTED** [review]
please address comments

File: tools/README.txt (hunk 1 above)
Comment (wdhawkins) [reply tools/README.txt]: Fixed.

**Review by ragusa — APPROVED** [review]

**Pull request merged**